lib/fluent/plugin/in_forward.rb

```diff
@@ -47,6 +47,8 @@ def initialize
     config_param :chunk_size_limit, :size, default: nil
     desc 'Skip an event if incoming event is invalid.'
     config_param :skip_invalid_event, :bool, default: false
+    desc "The field name of the client's hostname."
```

Comment:
It seems document issue like this: https://github.com/fluent/fluentd-docs/pull/215/files